app/test_main.py

```diff
@@ -1,3 +1,19 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (1, 1, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
```

Comment:
The expected results for the test cases do not seem to align with common conversion logic from animal years to human years. For example, a 1-year-old cat or dog typically corresponds to more than 0 human years. Please verify the logic in `get_human_age` to ensure it matches the expected outcomes in these test cases.